presto-main/src/main/java/io/prestosql/connector/system/jdbc/TableJdbcTable.java

```diff
@@ -81,8 +82,8 @@ public RecordCursor cursor(ConnectorTransactionHandle transactionHandle, Connect
     {
         Session session = ((FullConnectorSession) connectorSession).getSession();
         Optional<String> catalogFilter = tryGetSingleVarcharValue(constraint, 0);
-        Optional<String> schemaFilter = tryGetSingleVarcharValue(constraint, 1);
-        Optional<String> tableFilter = tryGetSingleVarcharValue(constraint, 2);
+        Optional<String> schemaFilter = tryGetSingleVarcharValue(constraint, 1).map(filter -> filter.toLowerCase(ENGLISH));
```

Comment:
Can we instead fix the check in the other place to allow non lower cased values? This seems invalid to me to manipulate values given in sql literals.